packages/client/src/clients/in-app-message/types.ts

```diff
@@ -84,7 +52,9 @@ export interface InAppMessageClientOptions {
   before?: string;
   after?: string;
   page_size?: number;
-  engagement_status?: InAppMessageEngagementStatus[];
+  engagement_status?:
+    | InAppMessageEngagementStatus
+    | InAppMessageEngagementStatus[];
```

Comment:
This seems whacky to me. @thomaswhyyou is this right?

Reply:
Yep! This is based on feedback from Thomas